tests/textcase/plugin/options/default_keymappings_false_spec.lua

```diff
@@ -0,0 +1,39 @@
+local textcase = require("textcase")
+
+local function get_buf_lines()
```

Comment:
TODO if this PR is merged: Create test helpers to share them across different files.

Reply:
Done already.